Description:
 Disable JavaRebelIntegration (no yet in Fedora and non-free stuff)
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2009-09-27
--- freemarker-2.3.23/source/build.xml
+++ freemarker-2.3.23/source/build.xml
@@ -207,6 +207,7 @@
       bootclasspath="${boot.classpath.j2se1.4}"
       excludes="
         freemarker/ext/beans/_EnumModels.java,
+        freemarker/ext/beans/JRebelClassChangeNotifier.java,
         freemarker/ext/dom/SunInternalXalanXPathSupport.java,
         
         freemarker/ext/jsp/**,
--- freemarker-2.3.23/source/ivy.xml
+++ freemarker-2.3.23/source/ivy.xml
@@ -116,8 +116,6 @@
     <dependency org="org.slf4j" name="jcl-over-slf4j" rev="&slf4j.version;" conf="build.base->default" />
     <dependency org="commons-logging" name="commons-logging" rev="1.1.1" conf="build.base->default" />
     
-    <dependency org="org.zeroturnaround" name="javarebel-sdk" rev="1.2.2" conf="build.base->default" />
-
     <dependency org="com.google.code.findbugs" name="annotations" rev="3.0.0" conf="build.base->default" />
     
     <!--dependency org="javax.script" name="script-api" rev="1.0" conf="build.base->default" /-->
